vendor/engines/secure_rooms/app/views/secure_rooms/facility_occupancies/show_problems.html.haml

```diff
@@ -0,0 +1,13 @@
+= content_for :h1 do
+  = current_facility
+
+= content_for :tabnav do
+  = render partial: "secure_rooms/shared/tabnav_occupancy",
+    locals: { secondary_tab: "problem" }
+
+- if @order_details.empty?
+  %p.notice= t(".no_problem_occupancies")
+- else
+  %p.notice= t(".instructions")
+  = render partial: "shared/problem_order_details",
+    locals: { assign_price_policies_path: assign_price_policies_to_problem_orders_facility_occupancies_path }
```

Comment:
I'm curious if we can do something like use `[:assign_price_policies_to_problem_orders, current_facility, Occupancy]` for the path and use something else to fill in the last item so we can get away without needing to pass anything in to `shared/problem_order_details`. Not a big deal, though.